pkg/graph/resolvers/plan_cr_tdl_test.go

```diff
@@ -0,0 +1,108 @@
+		JobCodeUSER:       true,
+		JobCodeASSESSMENT: true,
+	}
+	plan := suite.createModelPlan("My Plan")
+	dateInitiated := time.Now()
+	optionalComments := "My comments"
+
+	input := &model.PlanCrTdlCreateInput{
+		ModelPlanID:      plan.ID,
+		IDNumber:         "123-456",
+		DateInitiated:    dateInitiated,
+		Title:            "My CR",
+		OptionalComments: &optionalComments,
+	}
+
+	crTdl, err := PlanCrTdlCreate(suite.testConfigs.Logger, input, princ, suite.testConfigs.Store)
+
+	suite.NoError(err)
+	suite.EqualValues(crTdl.ModelPlanID, plan.ID)
+	suite.EqualValues(crTdl.IDNumber, "123-456")
+	suite.EqualValues(crTdl.Title, "My CR")
+	suite.EqualValues(crTdl.OptionalComments, optionalComments)
+
+}
+
+// TestPlanCrTdlUpdate tests updateing a plan_cr_tdl record in the database
+func (suite *ResolverSuite) TestPlanCrTdlUpdate() {
+
+	princ := &authentication.EUAPrincipal{
+		EUAID:             suite.testConfigs.UserInfo.EuaUserID,
+		JobCodeUSER:       true,
+		JobCodeASSESSMENT: true,
+	}
+	plan := suite.createModelPlan("My Plan")
+	crTdl := suite.createPlanCrTdl(plan, "123-456", time.Now(), "My CR", "My comments")
+
+	changes := map[string]interface{}{
+		"title":            "My new title",
+		"idNumber":         "654-321",
+		"optionalComments": "new comments",
+	}
+
+	result, err := PlanCrTdlUpdate(suite.testConfigs.Logger, crTdl.ID, changes, princ, suite.testConfigs.Store)
+
+	suite.NoError(err)
+	suite.EqualValues(crTdl.ID, result.ID)
+	suite.EqualValues(crTdl.ModelPlanID, result.ModelPlanID)
+	suite.EqualValues(result.Title, changes["title"])
+	suite.EqualValues(result.IDNumber, changes["idNumber"])
+	suite.EqualValues(result.OptionalComments, changes["optionalComments"])
+}
+
+// TestPlanCrTdlDelete tests deleting a plan_cr_tdl record in the database
+func (suite *ResolverSuite) TestPlanCrTdlDelete() {
+
+	princ := &authentication.EUAPrincipal{
+		EUAID:             suite.testConfigs.UserInfo.EuaUserID,
+		JobCodeUSER:       true,
+		JobCodeASSESSMENT: true,
+	}
+	plan := suite.createModelPlan("My Plan")
+
+	crTdl := suite.createPlanCrTdl(plan, "123-456", time.Now(), "My CR", "My comments")
+
+	del, err := PlanCrTdlDelete(suite.testConfigs.Logger, crTdl.ID, princ, suite.testConfigs.Store)
+	suite.NoError(err)
+	suite.EqualValues(del.ID, crTdl.ID)
+
+}
+
+// TestCrTdlGet returns a plan_cr_tdl record
+func (suite *ResolverSuite) TestPlanCrTdlGet() {
+	plan := suite.createModelPlan("My Plan")
+
+	crTdl := suite.createPlanCrTdl(plan, "123-456", time.Now(), "My CR", "My comments")
+
+	retCrTdl, err := PlanCrTdlGet(suite.testConfigs.Logger, crTdl.ID, suite.testConfigs.Store)
+	suite.NoError(err)
+	suite.EqualValues(retCrTdl.ID, crTdl.ID)
+
+}
+
+// TestPlanCrTdlsGetByModelPlanID tests returning all plan_cr_tdls associted with a plan
+func (suite *ResolverSuite) TestPlanCrTdlsGetByModelPlanID() {
+	plan := suite.createModelPlan("My Plan")
+
+	_ = suite.createPlanCrTdl(plan, "123-456", time.Now(), "My CR", "My comments")
```

Comment:
I genuinely am not sure what the standard is for ignoring return values in go, but you can also opt to entirely omit the `_ =` from these lines, and it'll work the same as using `_`

Not requesting that this be changed, but commenting to open a discussion about it! My thoughts:
- `_ = func()` makes it clear that there IS a return value, we just don't care about it
- `func()` is a bit more terse, maybe more readable?

Honestly, in writing this out, I've already started to lean more towards what you have here!

Reply:
I honestly just did `_=` because I saw it in some other tests 😅